### PR TITLE
Feature map vis

### PR DESCRIPTION
**Pull request by rubenwiersma**
Map of cameras is visualized. Only visualization is working right now, todo: updating database through map controls.

### CODE REVIEW

**Comment by cadw** [comment]
I have just added the database support, it now generates new cameras when clicking and updates existing cameras when dragging them across the screen.

**Comment by hungnguyen94** [comment]
Looks nice. Seems to work good. Isn't there another way to reload the canvas though? 
Perhaps that's something to look into in the future, as it works good now.

File: src/main/webapp/app/map/mapview.controller.js [hunk 1]
```diff
@@ -28,31 +30,74 @@
...
+                        var actualPosX = pointer.x;
+                        var actualPosY = pointer.y;
+
+                        var gridPosX = Math.floor(actualPosX / grid);
+                        var gridPosY = Math.floor(actualPosY / grid);
+
+                        var newCamera = new Object();
+                        newCamera.x = gridPosX;
+                        newCamera.y = gridPosY;
+                        newCamera.name = "New Camera";
```
Comment (hungnguyen94) [inline src/main/webapp/app/map/mapview.controller.js]: Maybe add a popup to define the name when creating.

**Comment by cadw** [comment]
I will add this next week, however, I think this can be merged now and as you stated everything works fine. So I think we can merge it right now and add this as a feature on another branch next week.

**Comment by hungnguyen94** [comment]
Alright, I'll merge it.

**Pull request merged**